From db71bb99da0f435d8d2df9aebf415f155872a17a Mon Sep 17 00:00:00 2001
From: Markus Niebel <Markus.Niebel@tq-group.com>
Date: Mon, 26 Nov 2018 18:05:44 +0100
Subject: [PATCH 063/188] arm: dt: imx6qdl-tqma6: use generic jedec,spi-nor

Allow different types, generic code can detect exact chip

restor behaviour from BSP 0110

Signed-off-by: Markus Niebel <Markus.Niebel@tq-group.com>
---
 arch/arm/boot/dts/imx6qdl-tqma6.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6qdl-tqma6.dtsi b/arch/arm/boot/dts/imx6qdl-tqma6.dtsi
index d61c5b84b179..70f78fd7c791 100644
--- a/arch/arm/boot/dts/imx6qdl-tqma6.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-tqma6.dtsi
@@ -36,7 +36,7 @@
 
 	flash: m25p80@0 {
 		status = "okay";
-		compatible = "micron,n25q128a13", "n25q128a13";
+		compatible = "jedec,spi-nor";
 		spi-max-frequency = <50000000>;
 		reg = <0>;
 		#address-cells = <1>;
